.github/gh-actions-self-hosted-runners/README.md

```diff
@@ -21,29 +21,29 @@ The current GitHub Actions workflows are being tested on multiple operating syst
 On the other hand, we will rely on GitHub-hosted runners for MacOS builds until a straightforward implementation approach comes out.
 
 ## Ubuntu
-Ubuntu Self-hosted runners are stored in Artifact Registry and implemented using Google Kubernetes Engine with the following specifications:
+Ubuntu Self-hosted runners are stored in Container Registry and implemented using Google Kubernetes Engine with the following specifications:
 
 #### Cluster
 * Cluster: [gh-actions-linux-runners](https://console.cloud.google.com/kubernetes/clusters/details/us-central1-a/gh-actions-linux-runners/details?project=apache-beam-testing)
-* Image: [linux-github-actions-runner](https://console.cloud.google.com/artifacts/docker/apache-beam-testing/us-central1/beam-github-actions/linux-github-actions-runner?project=apache-beam-testing)
+* Image: [linux-github-actions-runner](https://console.cloud.google.com/gcr/images/apache-beam-testing/global/ubuntu-runners-self-hosted?project=apache-beam-testing)
```

Comment:
just wondering is this change reflecting these containers are moving from artifact registry to container registry? per gcp documentation container registry is in maintenance mode and artifact registry supersedes it. There is some voice to migrate existing container workflows to artifact registry though not yet happened.